python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -89,6 +89,18 @@ def from_pretrained(cls,
         optimize_model = kwargs.pop("optimize_model", True)
 
         if load_in_4bit or load_in_low_bit:
+
+            if config_dict.get("quantization_config", None) is not None:
+                q_config = config_dict["quantization_config"]
+                if q_config["quant_method"] == "gptq":
+                    invalidInputError(q_config["bits"] == 4,
+                                      "Only 4-bit gptq is supported in bigdl-llm.")
+                    invalidInputError(q_config["desc_act"] is False,
+                                      "Only desc_act=False is supported in bigdl-llm.")
```

Comment:
also check `group_size`, group_size should be a multiple of 64.

Reply:
fixed